src/Data/RFC3339String/Format.purs

```diff
@@ -2,13 +2,30 @@ module Data.RFC3339String.Format where
 
 import Prelude
 
-import Data.Formatter.DateTime (FormatterCommand(..))
+import Data.DateTime.Locale (Locale(..))
+import Data.Formatter.DateTime (FormatterCommand(Placeholder, Milliseconds))
+import Data.Int (floor)
 import Data.List (List, fromFoldable)
 import Data.PreciseDateTime.Internal (dateTimeFormatISO)
+import Data.String (takeRight)
+import Data.Time.Duration (Minutes(..))
+import Math (abs)
 
 iso8601Format :: List FormatterCommand
 iso8601Format = dateTimeFormatISO <> fromFoldable
   [ Placeholder "."
   , Milliseconds
   , Placeholder "Z"
   ]
+
+-- Assumes the locale is valid, i.e. the offset is between
+-- [-720, +720] minutes.
```

Comment:
The spec says that offsets can range between `00:00` and `23:59`, so wouldn't that be ±1,400 minutes? (technically 1,439)

Reply:
I just assumed that timezones ranged from -12:00 to +12:00... comment changed.